apps/next_dev_portfolio/components/About.js

```diff
@@ -0,0 +1,25 @@
+import Button from "./Button";
+import { DotSmall } from "./Dot";
+
+export default function About() {
+  return (
+    <>
+      <div id="about"></div>
```

Comment:
I know I gave you this tip, but if your section has a margin-top anyway, you can just get rid of the empty element and instead of a margin, just give a padding-top to the section. It will be the same effect, without additional (maybe even confusing elements).